### PR TITLE
add separate logic for vcenter conv-snap

### PR DESCRIPTION
**Pull request by naumvd95**

### CODE REVIEW

File: servers/partner-ci/common.yaml [hunk 1]
```diff
@@ -341,8 +341,11 @@
                 ${SNAPSHOTS_URL/SNAPSHOTS_ID/$SNAPSHOTS_ID}
                 snapshots_id=$(grep "^CUSTOM_VERSION=" < snapshots.params | cut -d= -f2 | cut -d'#' -f2)
                 echo "snapshot_id_number=" $snapshots_id
-                python plugin_test/fuel-qa/utils/jenkins/conv_snapshot_file.py
-
+                if [ "${ENV_PREFIX}" == *"vcenter"* ]; then
```
Comment (rkhozinov) [inline servers/partner-ci/common.yaml]: [[ $SNAPSHOTS_ID == *"lastSuccessfulBuild"* ]]

File: servers/partner-ci/common.yaml [hunk 2]
```diff
@@ -341,8 +341,11 @@
                 ${SNAPSHOTS_URL/SNAPSHOTS_ID/$SNAPSHOTS_ID}
                 snapshots_id=$(grep "^CUSTOM_VERSION=" < snapshots.params | cut -d= -f2 | cut -d'#' -f2)
                 echo "snapshot_id_number=" $snapshots_id
-                python plugin_test/fuel-qa/utils/jenkins/conv_snapshot_file.py
-
+                if [[ "${ENV_PREFIX}" == *"vcenter"* ]]; then
```
Comment (rkhozinov) [inline servers/partner-ci/common.yaml]:
```
CONV_PATH="plugin_test/fuel-qa/utils/jenkins"
[[ "${ENV_PREFIX}" == *"vcenter"* ]] && CONV_PATH="utils/jenkins"
python "${CONV_PATH}/conv_snapshot_file.py"

```

**Review by rkhozinov — CHANGES_REQUESTED** [review]

**Comment by rkhozinov** [comment]
or you can use special variable in common_props to define path to the conv.py

**Comment by rkhozinov** [comment]
@naumvd95 see the comment ^

File: servers/partner-ci/common.yaml [hunk 3]
```diff
@@ -341,8 +342,7 @@
                 ${SNAPSHOTS_URL/SNAPSHOTS_ID/$SNAPSHOTS_ID}
                 snapshots_id=$(grep "^CUSTOM_VERSION=" < snapshots.params | cut -d= -f2 | cut -d'#' -f2)
                 echo "snapshot_id_number=" $snapshots_id
-                python plugin_test/fuel-qa/utils/jenkins/conv_snapshot_file.py
-
+                python "${CONV_SNAP_PATH}/conv_snapshot_file.py"
```
Comment (rkhozinov) [inline servers/partner-ci/common.yaml]: Let's define full path to the util:
`CONV_SNAP_PATH=utils/jenkins/conv_snapshot_file.py`

**Review by rkhozinov — CHANGES_REQUESTED** [review]

File: servers/partner-ci/common.yaml (hunk 3 above)
Comment (naumvd95) [reply servers/partner-ci/common.yaml]: done

**Review by rkhozinov — APPROVED** [review]

**Pull request merged**